README.md

```diff
@@ -491,6 +501,24 @@ select_distro() {
     done
 }
 
+# Function to unmount remaining mounted points
+unmount_chroot() {
+    local chmount="/data/local/chroot-distro/$selected_distro"
+    
+    for leftover in dev sys proc dev/pts sdcard system data \
+             storage/emulated storage/self \
+             "data/local/chroot-distro/$selected_distro/storage/emulated" \
+             "data/local/chroot-distro/$selected_distro/storage/self"; do
+        su -c "mount | grep -q '$chmount/$leftover'" >/dev/null 2>&1 && \
```

Comment:
There should not be a need to hide errors unless you are making too many assumptions about working conditions. This leads to sloppy coding, and instead you should ensure that during normal use you will not get errors.